pkgs/by-name/ad/adr-tools/package.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchurl, testers, adr-tools }:
+
+stdenv.mkDerivation rec {
+  pname = "adr-tools";
+  version = "3.0.0";
+
+  src = fetchurl {
+    url = "https://github.com/npryce/${pname}/archive/refs/tags/${version}.tar.gz";
+    hash = "sha256-lJDzGkV8JTxBEzE+1jUu/L+PkklwowmghIiDO5wyXXw=";
+  };
```

Comment:
You could change this to `fetchFromGithub`, but it does add a but more complexity. The `buildPhase` is a bunch of checks from upstream, and it expects to have `adr` in it's path. You would have to do your own `unpackPhase` like here: https://github.com/NixOS/nixpkgs/blob/5d31883bce72721ece5b3cbee89e2f22e79b240e/pkgs/tools/security/bmrsa/11.nix#L14. I think this is fine, though I would ask for a second opinion.

Reply:
fetchFromGitHub must be used here to make sure the correct tarball is downloaded